tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "useDefineForClassFields": true,
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
```

Comment:
Let's rename `DOM` to lowercase `dom` and the same for `dom.iterable`. I've seen issues arise before when these are upper cased like this.